tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -2,9 +2,6 @@
 
 namespace PHPStan\Analyser;
 
-use Bug4288\MyClass;
-use Bug4713\Service;
-use ExtendingKnownClassWithCheck\Foo;
```

Comment:
a few tests relied on importing classes, which can easily backfire when adding new tests, with same named classes.

therefore I adjusted `AnalyzerIntegrationTest` to use FQCN everywhere - inspired by `TypeCombinatorTest` where we do simliar stuff.